diff -up ndctl-64.1/configure.ac.orig ndctl-64.1/configure.ac
--- ndctl-64.1/configure.ac.orig	2019-03-22 15:46:50.206526000 -0400
+++ ndctl-64.1/configure.ac	2019-03-22 15:47:12.743488000 -0400
@@ -45,7 +45,7 @@ AM_CONDITIONAL([ENABLE_DOCS], [test "x$e
 AC_ARG_ENABLE([asciidoctor],
 	AS_HELP_STRING([--enable-asciidoctor],
 	[use asciidoctor for documentation build]),
-	[], enable_asciidoctor=yes)
+	[], enable_asciidoctor=no)
 AM_CONDITIONAL([USE_ASCIIDOCTOR], [test "x$enable_asciidoctor" = "xyes"])
 if test "x$enable_asciidoctor" = "xyes"; then
 	asciidoc="asciidoctor"
